contracts/src/v0.8/workflow/dev/v2/CapabilitiesRegistry.sol

```diff
@@ -542,15 +587,18 @@ contract CapabilitiesRegistry is INodeInfoProvider, OwnerIsCreator, ITypeAndVers
 
       if (node.encryptionPublicKey == bytes32("")) revert InvalidNodeEncryptionPublicKey(node.encryptionPublicKey);
 
-      bytes32[] memory capabilityIds = node.hashedCapabilityIds;
+      string[] memory capabilityIds = node.capabilityIds;
       if (capabilityIds.length == 0) revert InvalidNodeCapabilities(capabilityIds);
 
       ++storedNode.configCount;
 
       uint32 capabilityConfigCount = storedNode.configCount;
       for (uint256 j; j < capabilityIds.length; ++j) {
-        if (!s_hashedCapabilityIds.contains(capabilityIds[j])) revert InvalidNodeCapabilities(capabilityIds);
-        storedNode.supportedHashedCapabilityIds[capabilityConfigCount].add(capabilityIds[j]);
+        bytes32 hashedCapabilityId = _hash(capabilityIds[j]);
+        if (!s_hashedCapabilityIds.contains(hashedCapabilityId)) {
+          revert InvalidNodeCapabilities(capabilityIds);
+        }
+        storedNode.supportedHashedCapabilityIds[capabilityConfigCount].add(hashedCapabilityId);
```

Comment:
question on L628
https://github.com/smartcontractkit/chainlink-evm/pull/95/files#diff-52785757607154a33b9f84655208e21d5e3eb50daffc26b94724a01b106fe8c1R628

the i here is referring to `removedNodeP2PIds` is that the same index we want for 
`NodePartOfCapabilitiesDON(uint32(node.capabilitiesDONIds.at(i)), p2pId);`